src/base/emu-i386/simx86/codegen-x86.c

```diff
@@ -1875,6 +1875,10 @@ arith0:		{
 			G3M(0xc7,0x43,Ofs_DF_INCREMENTS,Cp);
 			G4(0xfcfeff,Cp);
 			break;
+		case CLI:
+			// andb $0xf8,EFLAGS+2(%%ebx)
+			G4M(0x80,0x63,Ofs_EFLAGS+2,0xf7u,Cp);
```

Comment:
Maybe use defined consts here instead of f7u?

Reply:
I'll send a 2nd commit to do that (should do defined consts for the others too then for consistency, i.e. CLC, STC, CMC, CLD and STD)